wikipedia-sentences/README.md

```diff
@@ -1,50 +1,76 @@
+
 # Semantic Wikipedia Search with Transformers and DistilBERT
 
 ![](https://docs.jina.ai/_images/jinabox-wikipedia.gif)
 
-| item   | content                                          |
-|--------|--------------------------------------------------|
-| Input  | 1 text file with 1 sentence per line             |
-| Output | *top_k* number of sentences that match input query |
 
-This is an example of using [Jina](http://www.jina.ai)'s neural search framework to search through a [selection of individual Wikipedia sentences](https://www.kaggle.com/mikeortman/wikipedia-sentences) downloaded from Kaggle. It uses the [`distilbert-based-uncased`](https://huggingface.co/distilbert-base-uncased) language model from [Transformers](https://huggingface.co).
+## Table of contents: 
+- [Overview](#overview)
+- [🐍 Build it yourself and deploy with Python](#---build-it-yourself-and-deploy-with-python)
+- [Overview of the files in this example](#overview-of-the-files-in-this-example)
+- [Flow diagram](#flow-diagram)
+- [Next steps, building your own app](#next-steps--building-your-own-app)
```

Comment:
Would look consistent to have emoticons for each heading

Reply:
will make a pr for the template